src/frontend/screens/Game/GamePage/index.scss

```diff
@@ -1,454 +1,27 @@
-        font-weight: 700;
-      }
-    }
-
-    .info .smallTitle {
-      font-weight: bold;
-    }
-
-    .reportProblem {
-      align-self: flex-start;
-      margin-top: 1.2em;
-      color: var(--accent);
-
-      &:hover {
-        color: var(--accent-overlay);
-      }
-
-      & svg {
-        margin-inline-end: 0.4em;
-      }
-    }
-  }
-
-  .gameRequirements {
-    align-self: flex-start;
-    overflow: auto;
-  }
-
-  .gamePageStoreIcon {
-    fill: var(--text-default);
-    flex-grow: 1;
-    text-align: end;
-    padding-inline-end: var(--space-sm);
-
-    & > svg {
-      width: var(--text-5xl);
-      height: var(--text-5xl);
-      padding: var(--space-3xs);
-      border-radius: 10px;
-    }
-  }
-
-  .MuiLinearProgress-root {
-    width: 169px;
-    height: 5px;
-    background-color: var(--neutral-05);
-
-    .MuiLinearProgress-bar {
-      background-color: var(--success);
-    }
-  }
-
-  .selectFieldWrapper.Field ~ .buttonsWrapper {
-    margin-top: 24px;
-  }
-
-  .updateText {
-    cursor: pointer;
-
-    &:hover {
-      color: var(--text-secondary);
-    }
-  }
-
-  .pictureTimeContainer {
-    display: flex;
-    flex-direction: column;
-    max-height: 400px;
-  }
-
-  .infoWrapper.itemContainer {
-    place-content: center;
-    text-align: initial;
-    width: 20vw;
-    min-width: fit-content;
-    max-width: 290px;
-    margin-bottom: 12px;
-  }
-
-  .itemTitle {
-    font-weight: var(--bold);
-    text-align: center;
-  }
-
-  .checkBox {
-    display: flex;
-    margin-top: 6px;
-    align-items: center;
-  }
-
-  .dlcTitle {
-    font-weight: var(--bold);
-    margin-top: 16px;
-  }
-
-  .sdlTitle {
-    cursor: pointer;
-    font-weight: var(--bold);
-
-    &:hover {
-      color: var(--download-button);
-    }
-  }
-
-  .MuiIconButton-colorPrimary,
-  .MuiCheckbox-colorPrimary.Mui-checked {
-    color: var(--download-button);
-  }
-
-  .PrivateSwitchBase-root-1 {
-    padding: var(--space-3xs);
-  }
-
-  .buttonsWrapper .button {
-    min-width: max-content;
-  }
-
-  .buttonWithIcon {
-    display: flex;
-    place-items: center;
-    justify-content: center;
-
-    & > svg {
-      height: 20px;
-      margin-inline-end: 5px; /* for compatibility, before was hardcoded in JS*/
-    }
-  }
-
-  @keyframes animate-stripes {
-    100% {
-      background-position: -100px 0px;
-    }
-  }
-}
-
 .App:not(.oldDesign) .gameConfigContainer {
   display: grid;
-  position: relative;
-  grid-template-columns: repeat(
-    2,
-    minmax(0, 1fr)
-  ); /* assure that the columns are always the same width */
-  grid-template-rows: min-content 1fr min-content;
-  row-gap: var(--space-md);
-  column-gap: min(var(--space-lg), 50px);
+  grid-template-columns: minmax(500px, 60%) minmax(500px, 60%);
+  padding: var(--space-md);
+  gap: var(--space-md);
+  border-radius: 10px;
+  max-width: 1920px;
+  max-height: 100vh;
   grid-template-areas:
-    'top top'
-    'main extra'
+    'header header'
+    'left right'
```

Comment:
if we can keep `main` and `extra` it makes it more semantic and makes more sense when we think about RTL layouts because in those cases the `main` ends up on the right side of the screen and the extra info ends up on the left side

![image](https://github.com/user-attachments/assets/7a3a7c7e-1b7e-4b0c-badd-82d51730658e)